packages/flutter_tools/gradle/module_plugin_loader.gradle

```diff
@@ -33,8 +35,20 @@ if (pluginsFile.exists()) {
     }
 }
 
+String flutterModulePath = project(':flutter').projectDir.parentFile.getAbsolutePath()
 gradle.getGradle().projectsLoaded { g ->
     g.rootProject.beforeEvaluate { p ->
+        p.subprojects { sp ->
+            if (object != null && object.plugins.android.name.contains(sanitizedPluginName)) {
```

Comment:
also check for `object.plugins != null && object.plugins.android != null`

Reply:
ok.